src/main/js/providers/ProviderTable.js

```diff
@@ -133,9 +134,11 @@ class ProviderTable extends Component {
     }
 
     retrieveData() {
-        const { descriptor } = this.state;
-        if (descriptor) {
-            this.props.getAllConfigs(descriptor.name);
+        const { descriptors, descriptorName } = this.props
+        const descriptorList = DescriptorUtilities.findDescriptorByNameAndContext(descriptors, descriptorName, DescriptorUtilities.CONTEXT_TYPE.GLOBAL)
+        if (descriptorList && descriptorList.length > 0) {
+            const foundDescriptor = descriptorList[0];
```

Comment:
The lines Paulo mentioned above is used through out the PR. The new function can be used here as well